docs/links/mautic_mail.py

```diff
@@ -0,0 +1,7 @@
+from . import link
+
+link_name = "Mautic Mail" 
+link_text = "Mautic Mail" 
+link_url = "security@mautic.org" 
```

Comment:
```suggestion
link_url = "mailto:security@mautic.org" 
```

The build is failing because this is an invalid link. I think it is expecting the `mailto:` directive just like you'd have `https://` in a link